Mopy/bash/game/skyrimvr/__init__.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# GPL License and Copyright Notice ============================================
+#  This file is part of Wrye Bash.
+#
+#  Wrye Bash is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  Wrye Bash is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with Wrye Bash; if not, write to the Free Software Foundation,
+#  Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+#
+#  Wrye Bash copyright (C) 2005-2009 Wrye, 2010-2015 Wrye Bash Team
+#  https://github.com/wrye-bash
+#
+# =============================================================================
+
+"""This modules defines static data for use by bush, when TES V:
+   Skyrim VR is set at the active game."""
+
+from .constants import *
+from ..skyrimse.default_tweaks import default_tweaks
+from ... import brec
+from ...brec import MreGlob
+from ..skyrimse import SkyrimSEGameInfo
+
+class SkyrimVRGameInfo(SkyrimSEGameInfo):
+    # This has to have a lowercase r because of displayname logic in bush...
+    displayName = u'Skyrim Vr'
+    fsName = u'Skyrim VR'
+    altName = u'Wrye Smash'
+    defaultIniFile = u'Skyrim_Default.ini'
+    exe = u'SkyrimVR.exe'
+    regInstallKeys = (
+        u'Bethesda Softworks\\Skyrim VR',
+        u'Installed Path'
+    )
+
+    vanilla_string_bsas = {
+        u'skyrim.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'update.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'dawnguard.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'hearthfires.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'dragonborn.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'skyrimvr.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa', u'Skyrim_VR - Main.bsa'],
+    }
+
+    @classmethod
+    def init(cls):
```

Comment:
I don't get why this needed copy and pasted. without it it crashes. According to some google searches `@classmethod`s are inherited.

😖